packages/docs/community/release-process/smart-contracts.md

```diff
@@ -34,26 +29,53 @@ Ongoing smart contract development is done on the `master` branch.
+
+## Build and Release Process
+Using the following scripts, new contracts can be built and deployed alongside a corresponding on-chain governance proposal. Run the following scripts from the branch with the new release contracts:
 
 ```bash
-yarn run release-contracts -d $CURRENTLY_RELEASED_BRANCH -f $FROM_ADDRESS -n $NETWORK
-```
+## Script 1 ##
+# verify the current contract bytecode on the network matches the contracts currently on the RC1 branch
+yarn truffle exec --network $NETWORK ./scripts/truffle/verify-bytecode.js --build_artifacts build/rc1 --before_release_1
+
+## Script 2 ##
+# generate report on backwards compatibility between RC1 and current branch with new release contracts
+yarn ts-node scripts/check-backward.ts report --exclude ".*Test|Mock.*|I[A-Z].*|.*Proxy|LinkedList|SortedLinkedList|SortedLinkedListWithMedian|MultiSig.*|ReleaseGold" -o build/rc1 -n build/contracts --output_file report.json
 
-This script does the following:
+## Script 3 ##
+# deploy new contracts and generate governance proposal for upgrade
+yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json'      
+```
 
-  1. Compiles the contracts at `$CURRENTLY_RELEASED_BRANCH` and confirms that the compiled bytecode matches what is currently deployed on the specified network.
-  2. Compiles the contracts in the current branch and checks backwards compatibility with what is currently deployed on the specified network, with the following exceptions:
+These scripts do the following:
+  #### Script 1:
+  1. Compiles the contracts at <code>rc1</code> branch and confirms that the compiled bytecode matches what is currently deployed on the specified network.
+  #### Script 2:
+  1. Compiles the contracts in the current branch and checks backwards compatibility with what is currently deployed on the specified network, with the following exceptions:
      1. If the STORAGE version has changed, does not perform backwards compatibility checks
      2. If the MAJOR version has changed, checks that the storage layout is backwards compatible, but does not check that the contract ABI is backwards compatible.
-  3. For contracts that have changed, confirms that the version number in the current branch is strictly greater than the deployed version number.
-  4. For contracts that have not changed, confirms that the version number in the current branch is exactly the same as the deployed version number
-  5. For contracts that have changed, deploys those contracts to the specified network.
-  6. Creates and submits a single governance proposal to upgrade to the newly deployed contracts.
+  2. For contracts that have changed, confirms that the version number in the current branch is strictly greater than the deployed version number.
```

Comment:
I'm not sure that script actually does these checks? I think it's `yarn verify-contracts` that does this

Reply:
not seeing this in `package.json`??

Reply:
I think @yorhodes might have insight on this stuff.